### PR TITLE
Enable management job notications for admins

### PR DESCRIPTION
**Pull request by jakemcdermott**
##### SUMMARY
Make the management job notification toggles enabled for admins. Relates to a bug in an external repo.

### CODE REVIEW

File: awx/ui/client/src/management-jobs/notifications/notification.controller.js [hunk 1]
```diff
@@ -19,6 +19,8 @@ export default
                 $scope[`${list.iterator}_dataset`] = Dataset.data;
                 $scope[list.name] = $scope[`${list.iterator}_dataset`].results;
 
+                console.log(isAdmin);
```
Comment (ryanpetrello) [inline awx/ui/client/src/management-jobs/notifications/notification.controller.js]: Intentional?

Comment (jakemcdermott) [reply awx/ui/client/src/management-jobs/notifications/notification.controller.js]: nope fixed thanks

File: awx/ui/client/src/management-jobs/notifications/notification.route.js [hunk 2]
```diff
@@ -39,6 +39,19 @@ export default {
             let path = `${GetBasePath('system_job_templates')}${$stateParams.management_id}`;
             Rest.setUrl(path);
             return Rest.get(path).then((res) => res.data);
+        }],
+        isAdmin: ['Rest', 'GetBasePath', function(Rest, GetBasePath) {
+            Rest.setUrl(GetBasePath('me'));
+            return Rest.get()
+                .then((res) => {
+                    if (res.data && res.data.results && res.data.count && res.data.results[0] && res.data.results.is_superuser) {
```
Comment (jlmitch5) [inline awx/ui/client/src/management-jobs/notifications/notification.route.js]: @nixocio _this_ is why ?. is so good, hah!

**Review by jlmitch5 — COMMENTED** [review]
looks good (nitpick, if there's a results[0] there should be a count, so I don't really know if that's super necessary, but I'm all for trying to guard this as much as possible so we don't have inadverdent bugs

**Review by jlmitch5 — APPROVED** [review]

File: awx/ui/client/src/management-jobs/notifications/notification.route.js (hunk 2 above)
Comment (marshmalien) [reply awx/ui/client/src/management-jobs/notifications/notification.route.js]: `res.data.results[0].is_superuser` ?

**Review by jlmitch5 — APPROVED** [review]

**Pull request merged**